resources/webroot/static/data/de.json

```diff
@@ -3220,6 +3285,8 @@
         "quest_30": "Erlöse %{amount} Pokemon",
         "quest_31": "Find %{amount} Team Rocket Invasion(en)",
         "quest_32": "%{amount} mal erster Grunt des Tages",
+        "quest_33": "Unbekannt",
```

Comment:
Whats quest 33 actually?
(And mind adding quest_33 and quest_34 in en.json also?

Reply:
33 = QUEST_BUDDY_FEED
So something like `Feed your buddy %{amount} times`
35, 36, 37 and 38 also excist tho